qa/tasks/mgr/dashboard/test_osd.py

```diff
@@ -100,7 +100,7 @@ def test_create_lost_destroy_remove(self):
     def test_safe_to_destroy(self):
         osd_dump = json.loads(self._ceph_cmd(['osd', 'dump', '-f', 'json']))
         unused_osd_id = max(map(lambda e: e['osd'], osd_dump['osds'])) + 10
-        self._get('/api/osd/{}/safe_to_destroy'.format(unused_osd_id))
+        self._get('/api/osd/safe_to_destroy?ids={}'.format(unused_osd_id))
```

Comment:
Hm I wonder, isn't there an API test you need to change here? If no one exits yet, it would be nice if you could create one :)

Reply:
I added a test for checking if multiple OSDs are safe to destroy.
There is no open issue for this problem.